concurrency/generators.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+)
+
+//implement generator by closure
```

Comment:
To comply with godoc conventions it should be `// FibnacciClosure implements fibonacci number geneartion` or something along those lines